spec/tasks/manage_admins_spec.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "rake admin:create", type: :task do
```

Comment:
I don't think I like testing rake tasks explicitly. It requires plenty of setup, and at times it looks like half the thing this is testing is "did we set up our rake task testing correctly?".

I guess this test is checking that we didn't mess up the order of arguments and that we get the domain from Rails config correctly, which is something. 

That said, well, it's set up and seems to pass, so I guess since we put in the work for it already we might as well keep it.